### PR TITLE
Feat/sui atom checkbox

### PR DESCRIPTION
**Pull request by a-ferrante**

### CODE REVIEW

**Review by jcger — APPROVED** [review]

**Comment by juanmaguitar** [comment]
Is there a paper for this new component? What is then the role of the types `RADIO` & `CHECKBOX` in `AtomInput` → https://sui-components.now.sh/workbench/atom/input/demo ?

Captura de pantalla 2019-06-27 a las 8 25 29

**Comment by a-ferrante** [comment]
Hi @juanmaguitar, here's the link to the UX-Spec document:
https://github.com/SUI-Components/UX-Definitions/blob/master/Atom/Checkbox-and-Radiobutton.md

I was not around when the discussion around inputs happened so i cannot provide input on the other question but the definition was already there, it just seems like the component wasn't implemented yet.

File: components/atom/checkbox/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$maw-atom-checkbox: 32px !default;
+
+.react-AtomCheckbox {
```
Comment (rmoralp) [inline components/atom/checkbox/src/index.scss:5]: All the sui-components has `sui` prefix. Why are you using `react` prefix?

**Comment by rmoralp** [comment]
Why look&feel does not match with paper?
Demo:
image

Paper:
image

File: components/atom/checkbox/src/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React, {Fragment} from 'react'
+import PropTypes from 'prop-types'
+
+const AtomCheckbox = ({id, value, disabled, checked, onChange, ...props}) => {
```
Comment (rmoralp) [inline components/atom/checkbox/src/index.js:4]: I think that onChange should be required or just has an empty arrow function as a default value

**Review by rmoralp — CHANGES_REQUESTED** [review]
@jcger will review with paper

**Comment by JoanClaret** [comment]
as @juanmaguitar  is pointing out, since this component is already developed i don't get why to develop it again instead of use or improve the existing one.

**Comment by a-ferrante** [comment]
Based on the discussions held i will close this PR and we will revisit the inclusion of this components later once Arturo is back.

**Pull request closed without merging**